syntax = "proto3";

package MetaOS;

service ShellControllerService {
    rpc ExecuteShell (stream ExecuteShellRequest) returns (stream ExecuteShellResponse) {}
}

message ExecuteShellRequest {
    string command = 1;
}

message ExecuteShellResponse {
    string output = 1;
    bool success = 2;
}